test/distributed/fsdp/test_fsdp_unshard_params.py

```diff
@@ -387,13 +387,16 @@ def _test_unshard_params_recurse(
         # Round up the sharded numel to account for padding
         sharded_inner_numel = int(math.ceil(unsharded_inner_numel / self.world_size))
         sharded_outer_numel = int(math.ceil(unsharded_outer_numel / self.world_size))
-        inner_flat_param = model.module[0]._handles[0].flat_param
-        outer_flat_param = model._handles[0].flat_param
+        inner_flat_param = model.module[0]._handle.flat_param
+        outer_flat_param = model._handle.flat_param
         self.assertEqual(sharded_inner_numel, inner_flat_param.numel())
         self.assertEqual(sharded_outer_numel, outer_flat_param.numel())
         expected_outer_numel = (
             unsharded_outer_numel if unshard_outer else sharded_outer_numel
         )
+        for fsdp_module in FSDP.fsdp_modules(fsdp_model):
```

Comment:
Did you insert this in the wrong place? (Should it be back to L607?)

(Test is failing since `fsdp_model` is not defined.)

Reply:
Yea I think so lol